api_v3/lib/KalturaDispatcher.php

```diff
@@ -157,6 +158,14 @@ public function dispatch($service, $action, $params = array())
 		
 		return $res;
 	}
+
+	protected function checkIfLoginIdAndTrim($service, $action, &$params)
```

Comment:
move to flag on the config, dont write explicit actions/services in the dispatcher